cli/cmd/install.go

```diff
@@ -542,6 +560,53 @@ func (options *installOptions) installPersistentFlagSet() *pflag.FlagSet {
 	return flags
 }
 
+// UpdateValuesFromConfig takes a values struct and updates its add-on values from the config installOption
+func (options *installOptions) UpdateAddOnValuesFromConfig(values *l5dcharts.Values) error {
+
+	if options.addOnConfig != "" {
+		addOnValues, err := ioutil.ReadFile(options.addOnConfig)
+		if err != nil {
+			return err
+		}
+
+		rawValues, err := yaml.Marshal(values)
+		if err != nil {
+			return err
+		}
+		// Merge Add-On Values with Values
+		finalValues, err := mergeRaw(rawValues, addOnValues)
+		if err != nil {
+			return err
+		}
+
+		if err = yaml.Unmarshal(finalValues, values); err != nil {
+			return nil
+		}
+
+	}
+
+	return nil
+}
+
+func mergeRaw(a, b []byte) ([]byte, error) {
+	aMap := make(map[string]interface{})
+	bMap := make(map[string]interface{})
+
+	err := yaml.Unmarshal(a, &aMap)
+	if err != nil {
+		return nil, err
+	}
+
+	err = yaml.Unmarshal(b, &bMap)
+	if err != nil {
+		return nil, err
+	}
+
+	chartutil.Values.MergeInto(aMap, bMap)
```

Comment:
This threw me off a bit, I had forgotten about this syntax in go :rofl: 
WDYT about using a more common idiom like this?
```go
var aMap, bMap chartutil.Values
...
aMap.MergeInto(bMap)
```